Tests/TuistSupportTests/Utils/GitControllerTests.swift

```diff
@@ -21,7 +21,7 @@ final class GitControllerTests: TuistUnitTestCase {
         let url = "https://some/url/to/repo.git"
         let path = try temporaryPath()
 
-        system.succeedCommand(["git", "-C \(path.pathString)", "clone \(url)"])
```

Comment:
Since we've touched how succeeded Command works, we've fixed every example where it is actually used.